scripts/modules/elastic_stack.py

```diff
@@ -1185,7 +1186,7 @@ def package_registry_url(options):
         return url
     if options.get("enable_package_registry"):
         return "http://package-registry:{}".format(PackageRegistry.SERVICE_PORT)
-    elif options.get("snapshot"):
+    elif options.get("snapshot") or not options.get("release"):
```

Comment:
@axw this changes the default to snapshot, which will serve snapshot+staging+production packages; just wanted to get your 👀 on this real quick.

Reply:
I suppose it's fine. My thinking was that we would rely on the logic in Kibana to select the appropriate registry, though to be fair we were still diverging by selecting the snapshot registry on snapshot stack images. But we can always select the staging repo if needed, so :+1: